contrib/network/connection_balance/dlb/test/config_test.cc

```diff
@@ -94,6 +118,20 @@ TEST_F(DlbConnectionBalanceFactoryTest, MakeFromDefaultProto) {
                              EnvoyException, HasSubstr("no available dlb hardware"));
 }
 
+TEST_F(DlbConnectionBalanceFactoryTest, MakeFromExactFallbackProto) {
+  envoy::config::core::v3::TypedExtensionConfig typed_config;
+  DlbConnectionBalanceFactory factory;
+  NiceMock<Server::Configuration::MockFactoryContext> context;
+
+  envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb dlb;
+  dlb.set_fallback_policy(
+      envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb::ExactConnectionBalance);
+  makeDlbConnectionBalanceConfig(typed_config, dlb);
+
+  EXPECT_LOG_CONTAINS("warn", "fallback to Exact Connection Balance",
+                      factory.createConnectionBalancerFromProto(typed_config, context));
```

Comment:
we should check the created connection balancer here is the exact balancer.
Can we likewise have a test with the NopFallbackProto?

Reply:
OK.

Reply:
@KBaichoo after `typeid().name()` test, I find that the result of `createConnectionBalancerFromProto()` is always `ConnectionBalancer` as its return value is `ConnectionBalancer` not the exact or nop balancer.